src/main/java/org/opensearch/sdk/BaseExtension.java

```diff
@@ -56,6 +56,8 @@ protected BaseExtension(String path) {
             }
         } catch (IOException e) {
             throw new ExceptionInInitializerError(e);
+        } catch (Exception e) {
+            throw new RuntimeException();
```

Comment:
I don't see the value in catching an exception and throwing it without any message.  Why not just make the previous catch block more general?

Reply:
I like that. Sure!